coda/coda_validate/views.py

```diff
@@ -172,7 +174,8 @@ def stats(request):
     # make a set of data that makes sense for the heatmap
     all_v = Validate.objects.all().order_by('last_verified')
     sums_by_date = {}
-    first_day, last_day = all_v[0], all_v.reverse()[0]
+    last_day = all_v[0], all_v.reverse()[0]
+    last_day = last_day[-1]
```

Comment:
This is ugly, but it's because you can't negative index this queryset. The second call is after the queryset evaluation has been triggered. Pushing a cleaner version of this. The two values don't need to be there at all -- it's a holdover from when an unused value was also set here.